snap.manifest.json

```diff
@@ -18,7 +18,9 @@
     }
   },
   "initialPermissions": {
-    "endowment:name-lookup": {},
+    "endowment:name-lookup": {
+      "chains": ["eip155:1", "eip155:11155111", "eip155:17000"]
```

Comment:
This is a copy of the ens supported chains found here: https://github.com/MetaMask/ens-resolver-snap/blob/3c6993c2738c8b7d4a413876291f311382ee3a4e/src/index.ts#L11